spec/clojure_ttt/board_spec.clj

```diff
@@ -29,8 +29,11 @@
   (it "should know if there is a draw"
     (should= true (draw? ["x"])))
 
-  (it "should know valid positions"
-    (should= false (valid-position? ["x"] 0)))
+  (it "should know if a position is valid"
+    (should= true (valid-position? ["x" "-"] 1)))
```

Comment:
I think you can use the function `should` and `should-not` rather than `should=` when things are just true/false